src/components/app/data/services/program.js

```diff
@@ -0,0 +1,97 @@
+import { getConfig } from '@edx/frontend-platform/config';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { camelCaseObject } from '@edx/frontend-platform';
+import { logError } from '@edx/frontend-platform/logging';
+import { getAvailableCourseRuns } from '../../../course/data/utils';
+import { fetchEnterpriseCustomerContainsContent } from './content';
+
+export async function fetchEnterpriseContainsContentItemProgram(enterpriseUuid, programUuid) {
```

Comment:
[inform] I believe you can use the general-purpose `queryEnterpriseCustomerContainsContent` now, here and below?